geocoding/example/lib/plugin_example/geocode_page.dart

```diff
@@ -1,12 +1,14 @@
 import 'package:flutter/material.dart';
 import 'package:geocoding/geocoding.dart';
-import 'package:geocoding_example/template/globals.dart';
+import '../template/globals.dart';
 
+/// Geocode example widget
 class GeocodeWidget extends StatefulWidget {
   @override
   _GeocodeWidgetState createState() => _GeocodeWidgetState();
 }
 
+/// Geocode example widget
```

Comment:
This comment is not necessary since the `_GeocodeWidgetState` is a private class. However if you do want to add documentation comment here it should adhere with the effective dart style guide (see more specifically rule "[PREFER starting library or type comments with noun phrases.](https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-library-or-type-comments-with-noun-phrases)").